### DISCUSSION TITLE
Is it possible hide agent in Chat UI

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#14416] [Q&A]
Hi
When I type “@” and an agent list pops up, I don’t want ‘workspace’ and ‘command’ to appear in the list. Is it possible to hide them?
![image](https://github.com/user-attachments/assets/27a78f59-f7f4-4991-9cf8-e668368300c8)

**Comment by JonasHelming** [comment]
You could just deactivate them in the AI Configuration View.
